src/pages/LocusPage.js

```diff
@@ -371,33 +374,67 @@ class LocusPage extends React.Component {
                 selectedIndexVariants,
                 selectedStudies,
               });
+              const isEmpty =
+                transformedData.geneTagVariants.length === 0 &&
+                transformedData.tagVariantIndexVariantStudies.length === 0;
+              const isEmptyFiltered =
+                filteredData.geneTagVariants.length === 0 &&
+                filteredData.tagVariantIndexVariantStudies.length === 0;
               const rows = locusTable(filteredData, lookups);
               return (
                 <React.Fragment>
-                  <LocusSelection
-                    {...{
-                      selectedGenes,
-                      selectedTagVariants,
-                      selectedIndexVariants,
-                      selectedStudies,
-                    }}
-                    lookups={lookups}
-                    handleDeleteGene={this.handleDeleteGene}
-                    handleDeleteTagVariant={this.handleDeleteTagVariant}
-                    handleDeleteIndexVariant={this.handleDeleteIndexVariant}
-                    handleDeleteStudy={this.handleDeleteStudy}
-                  />
-                  <Gecko
-                    data={filteredData}
-                    start={start}
-                    end={end}
-                    selectedGenes={selectedGenes}
-                    selectedTagVariants={selectedTagVariants}
-                    selectedIndexVariants={selectedIndexVariants}
-                    selectedStudies={selectedStudies}
-                    handleClick={this.handleClick}
-                    handleMousemove={this.handleMousemove}
-                  />
+                  <PlotContainer
+                    left={
+                      <BrowserControls
+                        handleZoomIn={this.handleZoomIn}
+                        handleZoomOut={this.handleZoomOut}
+                        handlePanLeft={this.handlePanLeft}
+                        handlePanRight={this.handlePanRight}
+                      />
+                    }
+                  >
+                    <PlotContainerSection>
```

Comment:
do the check of `isEmptyFiltered` outside `<PlotContainerSection></PlotContainerSection>`

```jsx
{isEmptyFiltered && <PlotContainerSection></PlotContainerSection>}
```

Then, you won't have to check inside the `PlotContainerSection` if it has children or not, keeping `PlotContainerSection` a simpler presentational component.